tasks/plugins.yml

```diff
@@ -25,5 +25,5 @@
   command: >
     java -jar {{ jenkins_jar_location }} -s http://{{ jenkins_hostname }}:{{ jenkins_http_port }}{{ jenkins_url_prefix | default('') }}/ install-plugin {{ item }}
     creates=/var/lib/jenkins/plugins/{{ item }}.jpi
-  with_items: jenkins_plugins
+  with_items: '{{ jenkins_plugins }}'
```

Comment:
I honestly don't know, I've only recently started using Ansible features like this. But I think it has always been supported. It's just that the non-quoted form is deprecated now,